Automattic/wp-calypso
# CRO pricing page i5: Add foldable FAQ's section

### Changes proposed in this Pull Request

This PR adds a new `FoldableFAQ` component to Calypso and displays a new FAQ section on the Plans page for Calypso (blue) and Cloud(green).

#### Implementation Notes
- I added the new FoldableFAQ component to the Calypso Docs, UI components section located here: 
- http://calypso.localhost:3000/devdocs/design
- Search for "`FoldableFAQ`".
- (You may need to restart your local Calypso build for this to work correctly.  There are some new files that need to be pulled into the build, I think)

### Testing instructions

- Download the PR
- Run both Calypso and Jetpack cloud concurrently
- Select the `i5 - Saas table design` variant of the a/b test `jetpackConversionRateOptimization`
- Visit the _Plans_ page
- In both platforms, check:
    - Verify that the FAQ section is there at the bottom of the page.
    - Verify that the FAQ's are functioning properly. Dropdowns are opening/closing etc..
    - Check that is matches the Figma design (Link in Asana)
    - Check that the new FAQ's are not showing for other a/b variants.

Fixes 1196341175636977-as-1199149328812610

![Screenshot on 2020-11-13 at 12-36-01](https://user-images.githubusercontent.com/11078128/99118731-e31f1600-25ac-11eb-8d00-bb6cb9206e42.png)
